src/compiler/scala/tools/nsc/backend/jvm/BCodeHelpers.scala

```diff
@@ -882,8 +882,8 @@ abstract class BCodeHelpers extends BCodeIdiomatic with BytecodeWriters {
       var fieldList = List[String]()
 
       for (f <- fieldSymbols if f.hasGetter;
-	         g = f.getter(cls);
-	         s = f.setter(cls);
+	         g = f.getterIn(cls);
+	         s = f.getterIn(cls);
```

Comment:
Oh, wow, that must have been a wrong string under my CTRL+V. Awesome you saw it. I will fix it as soon as I can. Do you happen to know how can I get sensible information from the tests? It seems just to fill my console, from what I can tell mostly with binary compatibility errors. I figured those were because I did not setup correctly.